src/main/scala/net/jcazevedo/moultingyaml/YamlPrinter.scala

```diff
@@ -0,0 +1,67 @@
+package net.jcazevedo.moultingyaml
+
+import org.yaml.snakeyaml.{DumperOptions, Yaml}
+
+/**
+  * @author Simone D'Avico (simonedavico@gmail.com)
```

Comment:
Very well, I'll remove it entirely. I'd rather have clean code than my name slapped everywhere.